app.js

```diff
@@ -0,0 +1,108 @@
+'use strict'
+
+var userName = prompt('What is your name?');
+    alert('It\'s nice to meet you ' + userName + '!');
+
+var firstQuestion = prompt('Was I born in the 1970s?');
+
+if (firstQuestion.toUpperCase() === 'YES' || firstQuestion.toUpperCase() === 'Y') {
+
```

Comment:
the first and last line of white space in a code block are unnecessary (eg. line 9 and 13 in this code block).